staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -227,12 +199,37 @@ func (s *store) conditionalDelete(ctx context.Context, key string, out runtime.O
 	}
 	for {
 		origState, err := s.getState(getResp, key, v, false)
-		if err != nil {
+		if err != nil && !storage.IsTransformerError(err) {
```

Comment:
also return if `origState` is nil, since we don't have the rev info to use later

Reply:
Added a return after the error checks.